PKHeX.Core/Resources/text/script/const_usum_es.txt

```diff
@@ -4,4 +4,4 @@
 434	Tapu Bulu	01:Luchable,02:Derrotado,03:Capturado
 404	Tapu Fini	03:Luchable,04:Derrotado,05:Capturado
 755	Zygarde	01:Luchable,02:Derrotado,05:Capturado
-867	Regalos de Tamaño-Tomtem 00:Ninguno,01:Uno Recibido,02:Dos Recibido,03:Tres Recibido,04:Cuatro Recibido,05:Cinco Recibido,06:Todos Recibidos
+867	Regalos de Tamaño-Tótem 00:Ninguno,01:Uno Recibido,02:Dos Recibido,03:Tres Recibido,04:Cuatro Recibido,05:Cinco Recibido,06:Todos Recibidos
```

Comment:
Totem Pokémon in Spanish are Pokémon Dominante, I would put "Dominantes por Regalo" or better, "Dominantes conseguidos con Dominsignias"; also I would swap "[Number] Recibido" to "Conseguido [Number]"